google-marketplace/test-marketplace.sh

```diff
@@ -10,7 +10,10 @@ fi
 REGISTRY=gcr.io/datadog-sandbox/datadog
 FULL_TAG=$1
 
+echo "Running gcrane cp"
 gcrane cp "datadog/operator:$FULL_TAG" "$REGISTRY/datadog-operator:$FULL_TAG"
-docker build --pull --no-cache --build-arg TAG="$FULL_TAG" --tag "$REGISTRY/deployer:$FULL_TAG" . && docker push "$REGISTRY/deployer:$FULL_TAG"
+echo "Running docker build"
+docker build --pull --platform linux/amd64 --no-cache --build-arg TAG="$FULL_TAG" --tag "$REGISTRY/deployer:$FULL_TAG" . && docker push "$REGISTRY/deployer:$FULL_TAG"
 # Note: do not use "createAgent": true, as when resources are cleaned up mpdev will orphan the DatadogAgent
-mpdev verify --deployer=$REGISTRY/deployer:"$FULL_TAG" --parameters='{"name": "datadog", "namespace": "datadog-agent", "datadog.global.credentials.apiKey": "00000000000000000000000000000000"}'
```

Comment:
the parameters were removed because they don't have any effect with the `mpdev verify` command.